admin_manual/configuration/server/occ_command.rst

```diff
@@ -0,0 +1,2132 @@
+
+Deleting a Setting
+~~~~~~~~~~~~~~~~~~
+
+Deleting a setting is quite similar to setting a setting. 
+In this case, you supply the username, application (or setting category) and key as above. 
+Then, in addition, you supply the ``--delete`` flag.
+
+::
+
+ sudo -u www-data php occ user:setting layla settings email --delete
+
+Generating a User Count Report
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Generate a simple report that counts all users, including users on external user authentication servers such as LDAP.
+
+::
+
+ sudo -u www-data php occ user:report
+ +------------------+----+
+ | User Report      |    |
+ +------------------+----+
+ | Database         | 12 |
+ | LDAP             | 86 |
+ |                  |    |
+ | total users      | 98 |
+ |                  |    |
+ | user directories | 2  |
+ +------------------+----+
+
+.. _syncing_user_accounts_label:
+
+Syncing User Accounts
+^^^^^^^^^^^^^^^^^^^^^
+
+This command syncs users stored in external backend services, such as *LDAP*, *Shibboleth*, and *Samba*, with ownCloud's, internal, user database.
+However, it's not essential to run it regularly, unless you have a large number of users whose account properties have changed in a backend outside of ownCloud.
+When run, it will pick up changes from alternative user backends, such as LDAP where properties like ``cn`` or ``display name`` have changed, and sync them with ownCloud's user database.
+If accounts are found that no longer exist in the external backend, you are given the choice of either removing or disabling the accounts. 
+
+.. note:: 
+   It's also :ref:`one of the commands <available_background_jobs_label>` that you should run on a regular basis to ensure that your ownCloud installation is running optimally.
+
+.. note::
+   This command replaces the old ``show-remnants`` functionality, and brings the LDAP feature more in line with the rest of ownCloud's functionality.
+
+Below are examples of how to use the command with an *LDAP*, *Samba*, and *Shibboleth* backend.
+
+LDAP
+~~~~
+
+::
+
+  sudo -u www-data ./occ user:sync "OCA\User_LDAP\User_Proxy"
+
+Samba
+~~~~~
+
+::
+
+  sudo -u www-data ./occ user:sync "OCA\User\SMB" -vvv
+
+Shibboleth
+~~~~~~~~~~
+
+::
+
+  sudo -u www-data ./occ user:sync "OCA\User_Shibboleth\UserBackend"
+
+Syncing via cron job
+~~~~~~~~~~~~~~~~~~~~
+
+Here is an example for syncing with LDAP four times a day on Ubuntu:
+
+  crontab -e -u www-data
+  
+  * */6 * * * /usr/bin/php /var/www/owncloud/occ user:sync -vvv --missing-account-action="remove" -n "OCA\User_LDAP\User_Proxy"
+ 
+.. _versions_label:
+ 
+Versions
+--------
+
+::
+
+ versions
+  versions:cleanup   Delete versions
+  versions:expire    Expires the users file versions  
+
+``versions:cleanup`` can delete all versioned files, as well as the ``files_versions`` folder, for either specific users, or for all users.
+The example below deletes all versioned files for all users::
+
+ sudo -u www-data php occ versions:cleanup
+ Delete all ver
```

Comment:
no timestamps in upgrade output in 9.0 and 9.1

Reply:
I don't follow what you're asking with this point.